config/default/common/config/wv.json/stories/severe-storm/atmospheric_rivers.json

```diff
@@ -0,0 +1,137 @@
+          "transition": {
+            "element": "",
+            "action": ""
+          },
+          "stepLink": "v=-149.2343104071579,26.61555273706992,-93.40895023395714,52.64027839533624&l=Reference_Features_15m,SMAP_L4_Analyzed_Root_Zone_Soil_Moisture&lg=true&l1=Reference_Features_15m,SMAP_L4_Analyzed_Root_Zone_Soil_Moisture&lg1=true&ca=true&t=2022-11-19-T23%3A35%3A22Z&t1=2023-04-06-T21%3A35%3A22Z"
+        },
+        {
+          "id": "008",
+          "description": "step008.html",
+          "transition": {
+            "element": "",
+            "action": ""
+          },
+          "stepLink": "v=-120.32664595033344,35.56312423104413,-119.07005472101878,36.370066631457775&l=Reference_Features_15m,Coastlines_15m,OPERA_L3_Dynamic_Surface_Water_Extent-HLS_Provisional(hidden,disabled=5-0-4),HLS_S30_Nadir_BRDF_Adjusted_Reflectance(hidden),HLS_L30_Nadir_BRDF_Adjusted_Reflectance&lg=true&l1=Reference_Features_15m,Coastlines_15m,OPERA_L3_Dynamic_Surface_Water_Extent-HLS_Provisional(disabled=5-0-4),HLS_S30_Nadir_BRDF_Adjusted_Reflectance(hidden),HLS_L30_Nadir_BRDF_Adjusted_Reflectance&lg1=true&ca=false&tr=atmospheric_rivers&t=2023-03-13-T05%3A35%3A22Z&t1=2023-04-30-T09%3A35%3A22Z"
+        },
+        {
+          "id": "009",
+          "description": "step009.html",
+          "transition": {
+            "element": "",
+            "action": ""
+          },
+          "stepLink": "v=-121.55756218602946,38.471499589321944,-120.70126583530936,38.96745403679784&l=Coastlines_15m,HLS_S30_Nadir_BRDF_Adjusted_Reflectance,HLS_L30_Nadir_BRDF_Adjusted_Reflectance&lg=true&l1=Coastlines_15m,HLS_S30_Nadir_BRDF_Adjusted_Reflectance,HLS_L30_Nadir_BRDF_Adjusted_Reflectance&lg1=true&ca=true&s=-119.751,36.0666&t=2022-11-20-T05%3A35%3A22Z&t1=2023-04-09-T09%3A35%3A22Z"
+        },
+        {
+          "id": "010",
+          "description": "step010.html",
+          "transition": {
+            "element": "",
+            "action": ""
+          },
+          "stepLink": "v=-122.06583991341911,39.245694471318245,-120.84924510836755,39.95032856655172&l=Coastlines_15m,HLS_S30_Nadir_BRDF_Adjusted_Reflectance,HLS_L30_Nadir_BRDF_Adjusted_Reflectance&lg=true&l1=Coastlines_15m,HLS_S30_Nadir_BRDF_Adjusted_Reflectance,HLS_L30_Nadir_BRDF_Adjusted_Reflectance&lg1=true&ca=true&s=-119.751,36.0666&t=2022-04-25-T20%3A12%3A08Z&t1=2023-04-20-T09%3A35%3A22Z"
+        },
+        {
+          "id": "011",
+          "description": "step011.html",
+          "transition": {
+            "element": "",
+            "action": ""
+          },
+          "stepLink": "v=-119.97132835257483,34.96174665313614,-119.55642237903686,35.18734014771676&l=Reference_Labels_15m,Coastlines_15m,HLS_S30_Nadir_BRDF_Adjusted_Reflectance,HLS_L30_Nadir_BRDF_Adjusted_Reflectance(hidden)&lg=true&l1=Reference_Labels_15m,Coastlines_15m,HLS_S30_Nadir_BRDF_Adjusted_Reflectance,HLS_L30_Nadir_BRDF_Adjusted_Reflectance&lg1=true&ca=true&t=2020-04-16-T14%3A00%3A00Z&t1=2023-04-11-T12%3A00%3A00Z"
```

Comment:
For step 11, the red superbloom on the right side of the map was partially cut off by the tour story widget in my browser.

Reply:
I zoomed this one out a bit too - it's going to be hard because they looked perfectly fine in my browser. For step 3 and step 11 I used the same extents as the atmospheric rivers video.